PROJECTS.md

```diff
@@ -0,0 +1,27 @@
+# What is this?
+
+This is a list of projects that the PR tech community has created to help in the hurricane Maria tech relief.
+
+All projects are owned by it's creators and many of them are donating the costs. As a community we want to help maintain these services until they are no longer needed. We want to cover the costs of these good samaritans.
+
+# How can I contribute?
+
+Take a look at the projects listed below their costs and the donations towards them will be listed. We suggest that if you want to make monetary contributions you look for projects that haven't reached their total goals.
+
+# Where does the money go?
+
+The money goes directly to the owners of the project or the person that is paying for the infrastructure fees.
+
+# How can I add my project?
+
+* Clone this repo
+* Add your project following the format of the ones before.
+   * Add your current cost
```

Comment:
If we are asking for a number, do we want to also ask for what the costs are specifically for, like hosting, diesel for generator, etc?

Reply:
So like a detail of cost then?

Reply:
That's what I was thinking. Doesn't have to be exact but can answer the question "What is my money paying for?"

Reply:
Yes. It should have details. If nothing else, for transparency when asking for funding.

Reply:
Excellent.